src/main/java/Event.java

```diff
@@ -0,0 +1,32 @@
+public class Event extends Task {
+    protected String at;
+
+    public Event(String des, String at) {
+        super(des);
+        this.at = at;
+    }
+
+    /**
+     * makes an event with the string given
+     * @param line input string that will be split into the task description and date
```

Comment:
I noticed this same coding violation across a few files. Perhaps it would be better to change these javadoc comments to fit the coding standards.